crates/ty_python_semantic/resources/mdtest/generics/legacy/classes.md

```diff
@@ -446,18 +454,18 @@ class C(Generic[T]):
     def generic_method(self, t: T, u: U) -> U:
         return u
 
-reveal_type(generic_context(C))  # revealed: tuple[T]
+reveal_type(generic_context(C))  # revealed: tuple[T@C]
 reveal_type(generic_context(C.method))  # revealed: None
-reveal_type(generic_context(C.generic_method))  # revealed: tuple[U]
+reveal_type(generic_context(C.generic_method))  # revealed: tuple[U@generic_method]
```

Comment:
Was thinking about whether it would be nice to show `U@C.generic_method`, but pyright also just shows `U@generic_method`, and there's a risk that `U@C.generic_method` might be misinterpreted as `(U@C).generic_method`? So.. probably best to leave everything as is.

Reply:
`U@generic_method` also aligns with [pyright's rendering](https://pyright-play.net/?strict=true&code=GYJw9gtgBALgngBwJYDsDmUkQWEMoDiApikSEgMYA0UAKokQGoCGIAsAFC1QC8dDLEAAoARLREBKTgFVe-BE1ajpkzhQA2zAM5aoAYSHFS5CgG1aAXQkAuTlHtQAJkWBQIRGAAswjoVqLqwDQArtaYKDASUAC0AHzhMLYcDilQIB7BIChQwXYOzq5oJGSUAPruXj5%2BAUGwYbQhYdJRcVDSSakO6TCZ2bkcnOkAbkTM6qXwCkJ6AHRFxmUV3o4SQA)